mne_bids/mne_bids.py

```diff
@@ -105,14 +106,9 @@ def _channels_tsv(raw, fname, overwrite=False, verbose=True):
 
     # get the manufacturer from the file in the Raw object
     manufacturer = None
-    if hasattr(raw, 'filenames'):
```

Comment:
is this fixed? I see that it is removed but doesn't really relate with the pr topic

Reply:
It does because now it should raise an error if `raw.filenames` does not exist ...